core/src/main/resources/hudson/slaves/Cloud/configure.jelly

```diff
@@ -37,11 +37,12 @@ THE SOFTWARE.
         <!-- main body of the configuration -->
         <st:include it="${instance}" page="config.jelly"/>
 
-        <l:isAdmin>
+        <l:hasAdministerOrManage>
```

Comment:
Why? Clouds can't be configured by manage currently?

https://github.com/jenkinsci/jenkins/blob/e41aeaacf434b9ff93c8848f8eddbfcbb27ed462/core/src/main/resources/jenkins/model/GlobalCloudConfiguration/index.groovy#L11-L12

Reply:
Could be discussed in another thread but I think cloud management could be lowered to `Jenkins.MANAGE`. In any case, looks like this isn't the place to change it.